kafka-ui-react-app/src/components/common/Pagination/__tests__/PageControl.spec.tsx

```diff
@@ -1,32 +1,45 @@
 import React from 'react';
-import { mount, shallow } from 'enzyme';
+import { shallow } from 'enzyme';
 import { StaticRouter } from 'react-router';
 import PageControl, {
   PageControlProps,
 } from 'components/common/Pagination/PageControl';
+import { ThemeProvider } from 'styled-components';
+import theme, { Colors } from 'theme/theme';
+import { screen } from '@testing-library/react';
+import { render } from 'lib/testHelpers';
 
 const page = 138;
 
 describe('PageControl', () => {
-  const setupWrapper = (props: Partial<PageControlProps> = {}) => (
-    <StaticRouter>
-      <PageControl url="/test" page={page} current {...props} />
-    </StaticRouter>
-  );
+  const setupComponent = (props: Partial<PageControlProps> = {}) =>
+    render(
+      <ThemeProvider theme={theme}>
+        <StaticRouter>
+          <PageControl url="/test" page={page} current {...props} />
+        </StaticRouter>
+      </ThemeProvider>
+    );
 
   it('renders current page', () => {
-    const wrapper = mount(setupWrapper({ current: true }));
-    expect(wrapper.exists('.pagination-link.is-current')).toBeTruthy();
+    setupComponent({ current: true });
+    expect(screen.queryByTestId('pagination-link')).toHaveStyle(
```

Comment:
We should use test-id only in **extreme** cases. Pls use role, text, label and other attrs